core/src/com/unciv/logic/city/CityStats.kt

```diff
@@ -499,8 +510,17 @@ class CityStats {
 
     private fun updateFoodEaten() {
         foodEaten = cityInfo.population.population.toFloat() * 2
-        if (cityInfo.civInfo.hasUnique("-50% food consumption by specialists"))
-            foodEaten -= cityInfo.population.getNumberOfSpecialists()
+        var foodEatenBySpecialists = 2f * cityInfo.population.getNumberOfSpecialists()
+        
+        foodEaten -= foodEatenBySpecialists
+        for (unique in cityInfo.civInfo.getMatchingUniques("-[]% food consumption by specialists")) 
+            foodEatenBySpecialists *= 1f - unique.params[0].toFloat() / 100f
+        
+        // Deprecated since 3.15
+            if (cityInfo.civInfo.hasUnique("-50% food consumption by specialists"))
+                foodEatenBySpecialists *= 0.5f
+        //
+        
+        foodEaten += foodEatenBySpecialists
```

Comment:
Same here. Maybe it's just me, I can see what you're doing but that's only because I know how the system works, to someone new this would look very strange.